src/qibolab/instruments/zhinst.py

```diff
@@ -491,7 +503,7 @@ def play(self, qubits, sequence, options):
 
         exp_dimensions = list(np.array(exp_res).shape)
         if dimensions != exp_dimensions:
-            logging.warning("dimensions %d , exp_dimensions %d", dimensions, exp_dimensions)
+            # logging.warning("dimensions %d , exp_dimensions %d", dimensions, exp_dimensions)
```

Comment:
Maybe
```suggestion
            logging.warning(f"dimensions {dimensions} , exp_dimensions {exp_dimensions")
```
works?

Reply:
It is suggested not to use f-strings, there's a dedicated lint in Pylint:
https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html

(there is written even the suggested way and the rational, so I'll avoid repeating)

Reply:
That way pylint will complain because it wants lazy formating of the string if I remenber correctly. I was planning to take a look today again.